src/UniqueVectors.jl

```diff
@@ -27,11 +28,11 @@ struct UniqueVector{T} <: AbstractUniqueVector{T}
     end
 end
 
-UniqueVector(items::Vector{T}) where {T} = UniqueVector{T}(items)
-UniqueVector(items::AbstractVector{T}) where {T} = UniqueVector{T}(Vector{T}(items))
-UniqueVector(items) = UniqueVector(collect(items))
+UniqueVector(items::AbstractVector{T}) where {T} = UniqueVector{T}(items)
+
+@deprecate UniqueVector(items) UniqueVector(collect(items))
```

Comment:
I can't recall at the moment why I marked this with a deprecation.  (Did I really mean to?)

Reply:
Oh yes, if `items` is not an `AbstractVector`, then we want to encourage the caller to specify the type by calling `UniqueVector{T}(items)`, else call `collect` explicitly.